eclipse-theia/theia
# Disable spellcheck in text input

#### What it does

Disable spellcheck of most input, including those DOM input elements

#### How to test

1. Open a input area in theia, such as dialog, file-dialog, file-upload-service, key-bindings-widget, mini-browser-content, 
preference-string-input, search-in-workspace-input, terminal-search-widget, vsx-extensions-search-bar
2. Type in something in the input area
3. Check whether there is spell check

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: elenmux <muxi.leng@ericsson.com>
